src/app/core/layout/content/content.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="page-area">
+  <h2>Contract Us</h2>
+  <p>For any kind of information please come and contract with us</p>
+  
+  <div class="row">
+    <div class="col-md-6">
+      <app-card>
```

Comment:
This is not a good example of a `shared component`, Please according to the previous suggestion from the demonstration. Additionally, you can visit the `wellday storybook` to have a better understanding.